arch/inst/Zcmop/c.mop.n.yaml

```diff
@@ -0,0 +1,39 @@
+# yaml-language-server: $schema=../../../schemas/inst_schema.json
+
+$schema: inst_schema.json#
+kind: instruction
+name: c.mop.n
+long_name: Compressed May-Be-Operation Instruction
```

Comment:
nit: could you remove "Instruction" from the name? It's a bit redundant, since the file is describing an instruction (see "kind" attribute, above). (Also, no other instruction YAML includes that word.)